### DISCUSSION TITLE
Add more themes

### DISCUSSION THREAD

**Discussion by iamrahatul** [iSoron/uhabits#658] [Feature Requests]
Hi. Thank you for this beautiful app.
**Would you mind adding more themes in this app?** Default colors look a bit dull to me.
You could use - _Dots, Dracula, Sakura, Monokai, Nord, Rama, Olivia_ and etc.

Here you can see what colors I'm talking about in the themes option - https://typings.gg

**Comment by iSoron** [comment]
Thank you for the suggestion, @iamrahatul. Pull requests adding more themes are welcome, although I am not planning to work on this myself.
